labs-ios-starter/Profiles/Favorites/FavoritesCollectionViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  FavoritesCollectionViewController.swift
+//  labs-ios-starter
+//
+//  Created by Alex Thompson on 2/2/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+private let reuseIdentifier = "CityCell"
+
+class FavoritesCollectionViewController: UICollectionViewController {
+    
+    let dummyFavorites: [Favorites] = [Favorites(city: "Southaven", state: "Mississippi"),
+                                  Favorites(city: "Memphis", state: "Tennessee"),
+                                  Favorites(city: "Olive Branch", state: "Mississippi")]
+    
+    @IBOutlet var cell: UICollectionViewCell!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    // MARK: UICollectionViewDataSource
+
+    override func numberOfSections(in collectionView: UICollectionView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
```

Comment:
Don't forget to remove this

Reply:
Copy that. I will get rid of these comments in the next pull. Thank you